packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -86,15 +88,17 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
   const isMeText = props.strings?.isMeText ?? strings.participantItem.isMeText;
   const menuTitle = props.strings?.menuTitle ?? strings.participantItem.menuTitle;
 
-  const avatarToUse = (
-    <Persona
-      text={displayName}
-      size={PersonaSize.size32}
-      presence={presence}
-      onRenderPersonaCoin={onRenderAvatar ? () => onRenderAvatar(props) : undefined}
-      className={mergeStyles(styles?.avatar)}
-      initialsTextColor="white"
-    />
+  const avatarOptions = {
+    text: displayName,
+    size: PersonaSize.size32,
+    presence: presence,
+    initialsTextColor: 'white'
+  };
+
+  const Avatar = onRenderAvatar ? (
```

Comment:
use of Capitalized name here is a bit weird. This is a component instance, not a Component definition.